textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/SpanTuple.java

```diff
@@ -0,0 +1,39 @@
+package edu.uci.ics.textdb.common.field;
+
+import java.util.List;
+
+import edu.uci.ics.textdb.api.common.Attribute;
+import edu.uci.ics.textdb.api.common.IField;
+
+public class SpanTuple extends DataTuple{
+    
+    private IField spanField;
+    private Span span;
+    private String key;
+
+    public SpanTuple(List<Attribute> schema, IField[] fields, 

```

Comment:
why it need `schema` and `fields` as passed in parameters?
It think the schema of this tuple is clear : `key, fieldname, span`, then the constructor could be like this:

```
SpanTuple(String key, IField spanField, Span span){
   super(new ArrayList<Attribute>(new StringField(), new StringField(), spanField), 
  new ArrayList<IField>(key, fieldName, span))
 ... 
}
```